pkgs/top-level/all-packages.nix

```diff
@@ -10850,9 +10850,12 @@ in
     inherit (darwin) libiconv libobjc libresolv;
   }) mx jvmci8 graalvm8;
 
-  inherit (callPackages ../development/compilers/graalvm/community-edition.nix {
+  graalvm-ce-pkgs = callPackages ../development/compilers/graalvm/community-edition.nix {
     inherit (darwin.apple_sdk.frameworks) Foundation;
-  }) graalvm8-ce graalvm11-ce;
+  };
+
+  graalvm8-ce = if stdenv.isLinux then graalvm-ce-pkgs.graalvm8-ce else null;
```

Comment:
That is not allowed because packages can never be null. Please mark it broken instead.

Reply:
even if linux works?
The change on this PR is that this new version of graalvm doesn't support darwin anymore for java8 but it supports for java11 and both for linux

Reply:
> even if linux works?

You can mark it broken on darwin or non linux, too.